Portfolio.html

```diff
@@ -0,0 +1,107 @@
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<link href="normalize.css" type="text/css" rel="stylesheet">
+	<link href="Portfoliostyle.css" type="text/css" rel="stylesheet">	
+	<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+
+
+</head>
+
+<body>
+	<nav>
+		<ul>
+			<li><a href="#home">Home</a></li>
+			<li><a href="#portfolio">Portfolio</a></li>
+			<li><a href="#aboutme">About Me</a></li>
+			<li><a href="#contact">Contact</a></li>
+		 </ul>
+	</nav>
+
+
+
+	<div class="intropage" id="home">
+		<div class="container">
+			<h1>JENNY ALLTON-TWI<b>S</b>T</h1>
+		</div>
+	</div>
+
+
+
+	<div class="portfolio" id="portfolio"><h2>Portfolio</h2>
+		<div class="container">
+			<div class="box">
+				<a href="#"><img src="tech.jpg" alt="Project Image"/></a>
+				<p>Some info about this very cool thing I invented here</p>
+				<h4>October 2020, Mayden Academy, Bath UK</h4>
+			</div>
+			<div class="box">
+				<a href="#"><img src="woman.jpg" alt="Project Image"/></a>
+				<p>Some info about this very cool thing I invented here</p>
+				<h4>October 2020, Mayden Academy, Bath UK</h4>
+			</div>
+			<div class="box">
+				<a href="#"><img src="hacker.jpg" alt="Project Image"/></a>
+				<p>Some info about this very cool thing I invented here</p>
+				<h4>November 2020, Mayden Academy, Bath UK</h4>
+			</div>
+			<div class="box">
+				<a href="#"><img src="handshake.jpg" alt="Project Image"/></a>
+				<p>Some info about this very cool thing I invented here</p>
+				<h4>December 2020, Mayden Academy, Bath UK</h4>
+			</div>
+			<div class="box">
+				<a href="#"><img src="lightbulb.jpg" alt="Project Image"/></a>
+				<p>Some info about this very cool thing I invented here</p>
+				<h4>January 2021, Bristol UK</h4>
+			</div>
+			<div class="box">
+				<a href="#"><img src="teamwork.png" alt="Project Image"/></a>
+				<p>Some info about this very cool thing I invented here</p>
+				<h4>January 2021, Bristol UK</h4>
+			</div>
+		</div>
+	</div>
+
+
+
+	<div class="aboutme" id="aboutme">
+		<div class="container">
+			<div class="medbox">
+				<h2>About Me</h2>
+				<img src="jennyphoto.jpg" alt="Jenny Allton-Twist looking happy"/>
+			</div>
+			<div class="medbox">
+				<p>Some stuff about me, a paragraph in which I talk humorously and succintly about my many achievements and accolades, as well as the interesting and altruistic activities I undertake in my spare time.</p>
+			</div>
+		</div>
+	</div>
+
+</body>
+
+
+<footer>
+	<div class="social">
+		<a href="https://twitter.com/jennyat9" class="fa fa-twitter"></a>
+		<a href="https://linkedin.com/in/jennyalltontwist" class="fa fa-linkedin"></a>
+		<a href="https://github.com/jennymegan" class="fa fa-github"></a>
+	</div>
+	<h6>Copyright J A-T 2020 &copy;</h6>
+</footer>
+
+
+
+<!-- Just in case you want to reinstate a dedicated Contact section -->
+
+<!--<div class="contact" id="contact">
```

Comment:
Even though it's not currently active I'd still put this ahead of footer as it kind of breaks document flow to have content below the footer.

Reply:
Cool, will do. Didn't know the convention :)